.gitignore

```diff
@@ -1,4 +1,3 @@
 /composer.lock
-/dist/
```

Comment:
Will be ignored in archives otherwise.

Reply:
Possible ramifications:

- https://github.com/elifesciences/api-raml/blob/develop/Jenkinsfile#L10 should not be affected
- https://github.com/elifesciences/api-raml/blob/develop/Jenkinsfile#L19 uses [`git add --all --force`](https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/elifeGitAutoCommit.groovy#L7) so should also not be affected

Reply:
RAMLifications.